packages/labs/task/src/task.ts

```diff
@@ -86,13 +104,15 @@ export type StatusRenderer<R> = {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export class Task<T extends [...unknown[]] = any, R = any> {
-  private _previousDeps: T = [] as unknown as T;
+  private _previousArgs: T = [] as unknown as T;
   private _task: TaskFunction<T, R>;
-  private _getDependencies: DepsFunction<T>;
+  private _args: ArgsFunction<T>;
   private _callId = 0;
   private _host: ReactiveControllerHost;
   private _value?: R;
   private _error?: unknown;
+  // Internal signal for being fully done processing a task, including host updates.
+  private _isIdle = true;
```

Comment:
Shouldn't idle always correspond to one of the existing TaskStatus's? Like `idle = this.status !== TaskStatus.PENDING`

Reply:
Removed.